### PR TITLE
zfcp-popup: timeout=0 is not enough for pressing key

### PR DESCRIPTION
**Pull request by Soulofdestiny**

### CODE REVIEW

**Comment by Soulofdestiny** [comment]
see: https://openqa.suse.de/tests/677914#step/releasenotes/4

File: tests/installation/releasenotes.pm [hunk 1]
```diff
@@ -23,7 +23,7 @@ sub run() {
     }
 
     # workaround for bsc#1014178
-    if (check_screen('zfcp-popup'), 0) {
+    if (check_screen('zfcp-popup'), 5) {
```
Comment (okurz) [inline tests/installation/releasenotes.pm]: The idea behind `check_screen(…), 0` was to assert that the screen is only checked as is at the time of call, no waiting. If you need to wait longer then maybe 5 seconds is also not enough. I recommend to either look for all alternatives with `assert_screen` and at least default timeout or the "lazy" approach:

```perl
wait_still_screen(3);
if (check_screen('zfcp-popup'), 0) {
…
}
```

**Comment by Soulofdestiny** [comment]
local verification:
http://opeth.suse.de/tests/4376#step/releasenotes/4

@okurz can you please review again?

File: tests/installation/releasenotes.pm [hunk 2]
```diff
@@ -23,7 +23,8 @@ sub run() {
     }
 
     # workaround for bsc#1014178
-    if (check_screen('zfcp-popup'), 0) {
+    wait_still_screen(5);
+    if (check_screen('zfcp-popup')) {
```
Comment (okurz) [inline tests/installation/releasenotes.pm]: no, that should be `check_screen('zfcp-popup', 0)`

Comment (Soulofdestiny) [inline tests/installation/releasenotes.pm]: http://opeth.suse.de/tests/4400#step/releasenotes/4

`15:31:03.0220 11830 <<< testapi::check_screen(mustmatch='zfcp-popup', timeout=30)
15:31:03.0826 11831 MATCH(releasenotes-zfcp-popup-20161207:1.00)
15:31:03.1490 11830 >>> testapi::_check_backend_response: found releasenotes-zfcp-popup-20161207, similarity 1.00 @ 382/373
15:31:03.1495 Debug: /var/lib/openqa/share/tests/sle/tests/installation/releasenotes.pm:36 called testapi::send_key
15:31:03.1496 11830 <<< testapi::send_key(key='alt-e')`

it continues directly with  alt-e

Comment (okurz) [inline tests/installation/releasenotes.pm]: Where does the 'alt-e' come from? Do you mean it immediately presses 'alt-o', this is not catched as the dialogue is not ready to accept that yet, then press 'alt-e'? I guess the `send_key 'alt-o';` should be guarded by

```perl
wait_screen_change { send_key 'alt-o' };
```

Comment (okurz) [inline tests/installation/releasenotes.pm]: problem is actually it's not `check_screen(…, 0)` but `check_screen(…), 0` which is nonsense. Please see https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/2253 first

**Comment by okurz** [comment]
see https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/2254

**Pull request closed without merging**